.github/workflows/tutor.yml

```diff
@@ -49,18 +54,19 @@ jobs:
         run: pip install -r requirements.txt
 
       - name: Generate tutor env
-        run:  tutor config save
+        run: |
+          docker buildx create --use
```

Comment:
This line must be in a separate step as it is not related to generating tutor env